docs: Add connection workflow docs (#4034)
hashicorp/boundary: website/content/docs/concepts/connection-workflows/workflow-ssh-proxycommand.mdx

---
layout: docs
page_title: SSH ProxyCommand
description: Learn how SSH ProxyCommand enables you to proxy an SSH connection in Boundary using a configuration file. Configure connections using the target's ID or domain.
---

# SSH ProxyCommand

> SSH `ProxyCommand` lets you proxy an SSH connection to a target according to a configuration file that you create.
> It can help simplify common developer and operator workflows.
> You can configure the file to connect to the target ID or the target domain.

Reviewer: It might be good to make this a little more succinct 

```suggestion
SSH `ProxyCommand` lets you proxy an SSH connection to a target to simplify common developer and operator workflows.
For example, you can configure the `.ssh/ssh_config` file to connect to the target ID or the target domain.
```
